tfx_addons/huggingface_pusher/README.md

```diff
@@ -99,6 +99,11 @@ HFSpaceConfig(
 ## Project Dependencies
 - [tfx](https://pypi.org/project/tfx/)
 - [huggingface-hub](https://pypi.org/project/huggingface-hub/)
+- [Git-LFS](https://git-lfs.github.com/) which could be installed as below
+```shell
+$ curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | sudo bash
+$ sudo apt-get install git-lfs
+```
```

Comment:
why is this needed?

Reply:
@casassg 

since the size of model file is large, HuggingFace Hub leverages Git-LFS feature to support it. So, I left a snippet to let people know that they need to include it in their Dockerfile.

Reply:
so this is not a dependency to use the library itself, but it is a dependency to push things to the remote HuggingFace Hub.

Reply:
interesting, is there no way to avoid it from having to be used?

If thats the case, I would recommend on Executor adding a short check that validates wether gitlfs exists and if not fails saying what needs to get added and how (aka point to HF pages)

Guessing HF code will already fail, but may be good to capture the error either before it fails there or capture the exception and parse it correctly for users of the component

Reply:
sounds reasonable! thanks for the advice, will do that